packages/media/src/createResource.js

```diff
@@ -58,7 +58,8 @@ import normalizeResourceSizes from './normalizeResourceSizes';
  * @property {boolean} isOptimized Whether the resource has already been optimized.
  * @property {boolean|null} isMuted Whether the resource has already been muted.
  * @property {Object} sizes Object of image sizes.
- * @property {?Object} output An optional object of video sizes for rendering gifs as videos
+ * @property {?Object} output An optional object of video sizes for rendering gifs as videos.
+ * @property {?Object} trimData An optional object of video trim data.
```

Comment:
Should we type this properly in JSDoc as well?

```
/**
 * @typedef {TrimData} Trim data object linking a trimmed video to its original
 * @property {number} original The ID of the original video.
 * @property {number} start This video's start offset relative to original video.
 * @property {number} end This video's end offset relative to original video.
 */
 ...
 * @property {?TrimData} trimData An optional object of video trim data.
```

Reply:
Just need to ensure the types are consistent with what we store in the database etc. Right now I think start/end is a string.